presentation/src/main/java/me/androidbox/presentation/login/viewmodel/LoginViewModel.kt

```diff
@@ -12,40 +9,56 @@ import kotlinx.coroutines.launch
 import me.androidbox.domain.authentication.ResponseState
 import me.androidbox.domain.authentication.model.Login
 import me.androidbox.domain.authentication.usecase.LoginUseCase
+import me.androidbox.presentation.login.screen.AuthenticationScreenEvent
+import me.androidbox.presentation.login.screen.AuthenticationScreenState
 import javax.inject.Inject
 
 @HiltViewModel
 class LoginViewModel @Inject constructor(
     private val loginUseCase: LoginUseCase
 ) : ViewModel() {
 
-    private val loginMutableState: MutableStateFlow<ResponseState<Login>?> = MutableStateFlow(null)
-    val loginState = loginMutableState.asStateFlow()
+    private val loginScreenMutableState = MutableStateFlow(AuthenticationScreenState<Login>("", "", false, "",null))
```

Comment:
Something I like to do, is have the default values in the State's class
So on AuthenticationScreenState I would add the default values there